pandas/_libs/lib.pyx

```diff
@@ -287,6 +277,7 @@ def fast_unique_multiple(list arrays, sort: bool = True):
         try:
             uniques.sort()
         except TypeError:
+            import warnings
```

Comment:
Really unsure about this specific change, the reason I made this change is to avoid going through the python space for no reason, we are not getting into the ```except``` block that often (that I know about).